src/components/Menu/Menu.tsx

```diff
@@ -66,13 +66,15 @@ const MenuButton = ({ children, className, ...other }: MenuButtonProps) => {
       status="neutral"
       {...other}
     >
-      {children}
-      <Icon
-        className={styles['menu__button--icon']}
-        name="expand-more"
-        purpose="decorative"
-        size="1.25rem"
-      />
+      <>
```

Comment:
I'm surprised we need a fragment here

Reply:
@ahuth this might be b/c headlessUI added warnings when nodes have more children than "they should". We still have some warnings that are false positives, so not sure how much actual value that has given us. :P

Reply:
Gotcha, thanks